install/installer/pkg/components/server/deployment.go

```diff
@@ -119,6 +122,17 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 				Name:  "WSMAN_CFG_MANAGERS",
 				Value: wsmanCfgManager,
 			},
+			{
+				Name: "no_grpc_proxy",
```

Comment:
@MrSimonEmms Shouldn't this be guarded by
```
if cfg.HTTPProxy == nil {
		return []corev1.EnvVar{}
	}
```
? :thinking:

Reply:
It's not strictly necessary as it'll still generate successfully (check the `cmd/testdata/render/minimal/output.golden` file as an example). But it is tidier not to have it in there so I'll change it